pombola/settings/base.py

```diff
@@ -23,6 +23,21 @@
 # switch on all debug when staging
 DEBUG          = STAGING
 
+if STAGING:
+    template_loaders = [
+        'django.template.loaders.app_directories.Loader',
+        'django.template.loaders.filesystem.Loader',
+        # 'django.template.loaders.eggs.Loader',
+    ]
+else:
+    template_loaders = [
+        ('django.template.loaders.cached.Loader', [
+            'django.template.loaders.app_directories.Loader',
+            'django.template.loaders.filesystem.Loader',
+            # 'django.template.loaders.eggs.Loader',
+        ]),
+    ]
+
```

Comment:
I'd possibly rewrite this as:
```
template_loaders = [
    'django.template.loaders.app_directories.Loader',
    'django.template.loaders.filesystem.Loader',
    # 'django.template.loaders.eggs.Loader',
]
if not STAGING:
    template_loaders = [('django.template.loaders.cached.Loader', template_loaders)]
```
…so that if you change the base loaders in future, you only have to change it in one location.